wiki/People/Quality_Assurance_Team/en.md

```diff
@@ -8,26 +8,26 @@ The **Quality Assurance Team**, commonly referred to as QAT, are the judges of q
 
 QAT members will be held to strict activity standards as their role requires them to be regularly involved in assessing beatmap quality.
 
-Members of this team will be promoted from [BNs](/wiki/People/Beatmap_Nomination_Group), should they accept the new role offered. Members of QAT have a red name in in-game chat and on the forums, whereas BNs no longer do.
+Members of this team will be promoted from [BN's](/wiki/People/Beatmap_Nominators/), should they accept the new role offered. Members of QAT have a red name on the in-game chat and on the forums, whereas BN's only have a purple name on the forums.
```

Comment:
possessive apostrophe why?

just change to Beatmap Nominators in all instances

Reply:
I agree that there shouldn't be apostrophes used, but there was a small discussion on acronyms and apostrophe in the server. Changing back.